packages/composite-checkout/src/components/checkout-payment-methods.tsx

```diff
@@ -131,6 +135,27 @@ export function CheckoutPaymentMethodsTitle() {
 	return <>{ ! isActive && isComplete ? paymentMethodLabelInactive : paymentMethodLabelActive }</>;
 }
 
+const GoogleDomainsCopyStyle = styled.p`
+	font-size: 14px;
+	color: ${ ( props ) => props.theme.colors.textColor };
+	margin: 0 0 16px;
+`;
```

Comment:
Copied from `ContactDetailsFormDescription` to match style across the steps.
![Screenshot 2023-07-31 at 15-03-48 Checkout — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/62062a2b-0cd9-4878-882c-199664067902)